src/main/java/ch/njol/skript/expressions/ExprExplodedBlocks.java

```diff
@@ -78,7 +194,7 @@ public boolean isSingle() {
 	
 	@Override
 	public String toString(@Nullable Event e, boolean d) {
-		return "exploded blocks";
+		return e instanceof EntityExplodeEvent ? "entity" : "block" + " event's exploded blocks";
```

Comment:
You shouldn't use the event here:
```suggestion
		return "exploded blocks";
```
toString is usually called when the event argument is null, because it's called during parsing

Reply:
Good to know, sure :)